myo_multithreading_examp.py

```diff
@@ -41,4 +42,4 @@ def print_battery(bat):
 
 	except KeyboardInterrupt:
 		print("Quitting")
-		quit()
\ No newline at end of file
+		quit()
```

Comment:
this is a test 
The software constructs all or part of an SQL command using externally-influenced input from an upstream component, but it does not neutralize or incorrectly neutralizes special elements that could modify the intended SQL command when it is sent to a downstream component.

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Readable

Reply:
smoke test 12/9/2021



---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Not secure, Inefficient, Reusable

Reply:
---
**Sema Tags:** Readable, Secure, Efficient, Elegant, Reusable, Fault-tolerant

Reply:
cool!

not secure 

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not secure

Reply:
test 12/22/2021

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant, Not reusable, Fault-tolerant

Reply:
always true
Make the code a little bit less clever in order to make it easier to read. Give it a stronger narrative structure so that you can more easily understand the code.

📄 [Sema Community](none)

Logic should in general be asking the question "Is this true?" instead of "Is this not true?"

📄 [Principles.dev](https://principles.dev/p/)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix